draft-wussler-openpgp-pqc.md

```diff
@@ -517,9 +545,9 @@ Even though the zero point, also called the point at infinity, may occur as a
 result of arithmetic operations on points of an elliptic curve, it MUST NOT
 appear in any ECC data structure defined in this document.
 
-Furthermore, when performing the explicitly listed multiplications in
-{{x25519-x448-kem}} or {{ecdh-kem}} it is REQUIRED to perform all checks,
-clamping, or masking mandated from the relative elliptic curve specification.
+Furthermore, when performing the explicitly listed operations in
+{{x25519-kem}}, {{x448-kem}} or {{ecdh-kem}} it is REQUIRED to follow the
+security advisory mandated from the relative elliptic curve specification.
```

Comment:
I don't think "security advisory" is entirely covering what is intended to be conveyed by the previous formulation. It would rather say:
```suggestion
specification and security advisory mandated from the relative elliptic curve specification.
```